kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -39,49 +39,42 @@
         :layout8="{ span: 6 }"
         :layout12="{ span: 10 }"
       >
-        <TabsWithOverflow
+        <KListWithOverflow
           tabsId="quizSectionTabs"
           class="section-tabs"
-          :tabs="tabs"
-          :activeTabId="activeSection ?
-            activeSection.section_id :
-            '' "
-          backgroundColor="transparent"
-          hoverBackgroundColor="transparent"
-          :aria-label="quizSectionsLabel$()"
-          @click="id => setActiveSection(id)"
+          :items="tabs"
         >
-          <template #tab="{ tab }">
+          <template #item="{ item }">
             <span
-              :ref="tabRefLabel(tab.id)"
+              :ref="tabRefLabel(item.id)"
               appearance="flat-button"
               style="display: inline-block;"
               :appearanceOverrides="tabStyles"
             >
-              {{ tab.label }}
+              {{ item.label }}
             </span>
           </template>
 
-          <template #overflow="{ overflowTabs }">
+          <template #more="{ overflowItems }">
             <KIconButton
-              v-if="overflowTabs.length"
+              v-if="overflowItems.length"
               tabindex="-1"
               class="overflow-tabs"
               icon="optionsHorizontal"
-              :style="overflowButtonStyles(overflowTabs)"
+              :style="overflowButtonStyles(overflowItems)"
```

Comment:
I wonder if the `overflowItems` given by the new `KListWithOverflow` are the same shape as the data we were getting before because of the missing styles -- however, this may only be due to the missing `KTabsList` component issue mentioned above.